src/main/java/com/pi4j/example/components/ADS1115.java

```diff
@@ -82,7 +82,7 @@ public class ADS1115 extends Component {
     /**
      * old value from last successful read of conversion register (raw data)
      */
-    private int[] oldValue = new int[4];
+    private final int[] oldValue = new int[4];
```

Comment:
why final? the value from the array is changing all the time. Or is final regarding the size of the int array and not the value in each index